src/playground/components/Alert.js

```diff
@@ -70,12 +70,26 @@ export default function Alert({ type, text, message, onFix }) {
                     </ul>
                 </>
             ) : (
-                fix && (
-                    <button onClick={() => onFix(message)} className="alert__fix-btn">
+                <div style={{
+                    display: "flex"
+                }}>
```

Comment:
```suggestion
                <div className="alert__actions">
```
Pls, avoid inline style. Create a class and add styles to that class.

In the `alert.scss` style file:
```
.alert__actions {
 display: flex:
}
```